docs/edge.rst

```diff
@@ -456,13 +456,133 @@ Cloud Console overview <cloud-reference:overview-org-regions>`.
 With this, you should be ready to use CrateDB Edge via Microk8s.
 
 
+.. _edge-tools-k3s:
+
+K3S
+---
+
+Below is a full walkthrough of how to get CrateDB Edge up and running on K3S.
+The steps are merely examples of a process validated by us; other methods may
+work also. We provide this information for ease of use and to illustrate how to
+work with CrateDB Edge.
+
+
+Set up K3S
+''''''''''
+
+A simple way to bootstrap the K3S setup is with `k3sup`_. However, this
+tutorial assumes you will use K3S native, which offers more granularity. Also,
+this setup is suitable for a multi-node cluster.
+
+First you have to set up the master node:
+
+.. code-block:: console
+
+    export INSTALL_K3S_VERSION="v1.19.10+k3s1"
+    curl -sfL https://get.k3s.io | sh -s - --disable=traefik
+
+    mkdir ~/.kube
+    cp /etc/rancher/k3s/k3s.yaml ~/.kube/config
+    export KUBECONFIG=~/.kube/config
+    kubectl config set-context default
+    kubectl get node -o wide
+
+Next, get the token:
+
+.. code-block:: console
+
+    cat /var/lib/rancher/k3s/server/node-token
+
+Note that the master node will operate both as a master and as a worker.
+
+
+Join nodes to cluster
+'''''''''''''''''''''
+
+Next, you set up other worker nodes (as many as applicable to your use case):
+
+.. code-block:: console
+
+    export token=<token>
+    export INSTALL_K3S_VERSION="v1.19.10+k3s1"
+    curl -sfL https://get.k3s.io | K3S_URL="https://ub1:6443" K3S_TOKEN=$token sh -
+
+
+Uninstall
+'''''''''
+
+If you need to uninstall, run:
+
+.. code-block:: console
+
+    /usr/local/bin/k3s-agent-uninstall.sh
+
+
+Use a storage solution
+''''''''''''''''''''''
+
+The Microk8s setup will require a storage solution. In this case, the tutorial
```

Comment:
```suggestion
Our CrateDB Edge setup will require a storage solution. In this case, the tutorial
```

Reply:
That's what I get for lazy copy-pasting, well caught